### ISSUE TITLE
Isssue after running update.sh

### ISSUE THREAD

**Issue by MichaelDR2016** [jamiegau/cinema-catcher-app#10]
After running update.sh I am getting an error when its trying to start the containers. On the containers backendc and backend I get 
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failedd:unable to start container process: exec: "/usr/bin/gunicorn": stat /usr/bin/gunicorn: no such file or directory: unknown

**Comment by jamiegau** [comment]
This will be because there has been some changes to the docker-compose.yml file.

I also moved from using a local.env.sh to putting those VAR settings into the .env file.   Please make sure that is the case as is explained in the updated install instructions.

I recommend doing a
`git reset --hard HEAD`
then
`git pull`

This should force an update to the latest docker-compose.yml file.
see #7 for the main reason for the update.

**Comment by MichaelDR2016** [comment]
This worked. Thank you. Do you have a discord? I have made some things to automate our theatre and I wanted to see if you would want any of it to add into the app

**Comment by jamiegau** [comment]
I don't use Discord much.  I have enough distractions already.
Post idea on here,  I'll see them.

**Comment by MichaelDR2016** [comment]
I don't have a ton of ideas; it's mainly just things I've implemented for our theatre. We're a Dolby house. The setups I've completed so far include:

Automatic Ingests: From house to NAS.
Notifications via Discord: For well, anything.
Power Management: We power down our machines at night and power them up every morning. The power-on is done with Wake-on-LAN, along with a script to turn on projectors simultaneously. Ours are Christie machines.
Automatic Scheduler: I've developed a scheduler that runs and sets schedules up to a specified end date. Currently, I'm focusing on this mostly. It alerts if there's no playlist for a session but doesn't do a great job at ensuring there are no conflicts in the schedule. I still have to manually create and distribute the playlists.
Drive Completion Notifications: I receive alerts when new drives I am ingesting are complete, so I know when to swap in a new one.
Software update notifications: I receive discord messages when say theres a firmware update for our doremi Showvaults. 
Content Ingestion Scripts: I have a few scripts that work together to help me ingest content. One of them goes through newly plugged-in drives and sorts them, making it much easier to identify what's what and filter out unnecessary content, such as Atmos or 7.1, as we don't use captions or other languages. Most of this is automated.

I am still new to how everything works. I've been into tech and programming and took over all the tech debt from my theatre about six months ago. I've been looking at this project for a bit and have tinkered with it off and on for a few months but haven't yet migrated anything into or with it. I'm not entirely even a back of house person, just someone who came into management and saw the opportunity to learn and maybe make some stuff. Take on some more responsibility and just automate everything I can. Discord has been one of the best things for me. Our managers can use commands to interact with everything. Most of our managers are older, and very tech illiterate so turning off a bulb on a theatre that didn't sell somehow is too much, but !off 3 in discord to turn it off is easy enough. Let me know if you want any of what I've written to try to use it as a base to add anything or anything like that
